Sources/Client/Client/ClientLogger.swift

```diff
@@ -128,6 +128,10 @@ public final class ClientLogger {
             return intersectedOptions.isEnabled ? ClientLogger(icon: icon, level: .level(intersectedOptions)) : nil
         }
     }
+  
+    /// Controls whether connection error dialogue is shown on WebSocket errors.
+    /// you can check `showConnectionErrorAlert` function to see how the dialogue is being shown.
+    public static var showConnectionErrorAlert = false
```

Comment:
Why did you decider to change it to `false` when the current default is `true`?

Reply:
Same reason as showing emoji in logging: people didn't want this on by default